master/buildbot/test/unit/test_secret_in_passwordstore.py

```diff
@@ -0,0 +1,102 @@
+# This file is part of Buildbot.  Buildbot is free software: you can
+# redistribute it and/or modify it under the terms of the GNU General Public
+# License as published by the Free Software Foundation, version 2.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc., 51
+# Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Copyright Buildbot Team Members
+
+import subprocess
+from pathlib import Path
+from unittest.mock import PropertyMock
+from unittest.mock import patch
+
+from twisted.internet import defer
+from twisted.python.filepath import FilePath
+from twisted.trial import unittest
+
+from buildbot.secrets.providers.passwordstore import SecretInPass
+from buildbot.test.util.config import ConfigErrorsMixin
+
+
+class TestSecretInPass(ConfigErrorsMixin, unittest.TestCase):
+
+    def createTempDir(self, dirname):
+        tempdir = FilePath(self.mktemp())
+        tempdir.createDirectory()
+        return tempdir.path
+
+    @defer.inlineCallbacks
+    def setUp(self):
+        with patch.object(Path, "is_file", return_value=True):
+            self.tmp_dir = self.createTempDir("temp")
+            self.srvpass = SecretInPass("password", self.tmp_dir)
+            yield self.srvpass.startService()
+
+    @defer.inlineCallbacks
+    def tearDown(self):
+        yield self.srvpass.stopService()
+
+    def testCheckConfigSecretInPassService(self):
```

Comment:
Can we use `test_check_config_secret_in_pass_service` naming? Over 90% of buildbot tests use this approach.

Reply:
Sure, I went with camel because the file provider test used that case, I prefer snake in Python anyway!